components/dashboard/src/settings/EnvironmentVariables.tsx

```diff
@@ -222,7 +222,17 @@ export default function EnvVars() {
             <div className="flex items-start sm:justify-between mb-2">
                 <div>
                     <h3>Environment Variables</h3>
-                    <h2 className="text-gray-500">Variables are used to store information like passwords.</h2>
+                    <h2 className="text-gray-500">
+                        Variables are used to store information like passwords.{" "}
+                        <a
+                            className="gp-link"
+                            href="https://www.gitpod.io/docs/configure/projects/environment-variables#environment-variables"
```

Comment:
**Thought (un-related to PR):** We need to break this env variables docs page down, not all the environment variables that are described on that page are related to the environment variables set here. Hierarchically the content is also under "project" right now (as per the URL) but really this should be `/user/` for this page.